test/CoveragePool.test.js

```diff
@@ -61,20 +61,275 @@ describe("CoveragePool", () => {
     await assetPool.connect(underwriter).deposit(to1e18(400))
   })
 
+  describe("beginRiskManagerApproval", () => {
+    context("when caller is not the governance", () => {
+      it("should revert", async () => {
+        const notGovernance = await ethers.getSigner(5)
+        await expect(
+          coveragePool
+            .connect(notGovernance)
+            .beginRiskManagerApproval(riskManager.address)
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when caller is the governance", () => {
+      beforeEach(async () => {
+        await coveragePool
+          .connect(governance)
+          .beginRiskManagerApproval(riskManager.address)
+      })
+
+      it("should not approve risk manager", async () => {
+        expect(await coveragePool.approvedRiskManagers(riskManager.address)).to
+          .be.false
+      })
+
+      it("should store risk manager approval timestamp", async () => {
```

Comment:
`approval process begin timestamp` perhaps? Same for the other, similar tests.

Reply:
I changed it to "should store approval process begin timestamp".